servo-tidy.toml

```diff
@@ -88,20 +88,27 @@ files = [
   "./components/style/counter_style/mod.rs",
   "./components/style/properties/declaration_block.rs",
   "./components/style/properties/helpers.mako.rs",
+  "./components/style/queries/feature_expression.rs",
```

Comment:
At this point, it might make more sense to simply ignore all of `components/style` and the rest of the crates that we vendor from Gecko.